code-challenges.test.js

```diff
@@ -69,14 +180,70 @@ var middleLetters2 = "rhinoceros"
 
 // b) Create the function that makes the test pass.
 
-
+//write a function that takes a string as an argument
+const findMdlLet = (str) => {
+    //create a variable to store the position
+    var posOfMdl;
+    //create a variable to store the length
+    var lengthOfMdl;
+    //write an if statement to check if the string length is odd number of characters
+    if(str.length % 2 == 1) {
+        //if odd return the position by dividing the length of string by two 
+        posOfMdl = str.length / 2;
+        //and return the lengthOfMdl of 1 becuase there is only one middle letter
+        lengthOfMdl = 1;
+    } else {
+        //else return the position by dividing the length by 2 and subtracting 1
+        posOfMdl = str.length / 2 - 1;
```

Comment:
```suggestion
        posOfMdl = str.length;
```
because it's always dividing by 1